cv_next/app/hall/page.tsx

```diff
@@ -1,54 +1,81 @@
+import Image from "next/image";
 import { ScrollToTop } from "@/components/ui/scrollToTop";
 import { External_Credits as contributors } from "@/lib/definitions";
 
 const hall: React.FC = () => {
   return (
-    <div className="relative mx-auto h-full w-[700px] max-w-full xl:w-[1400px]">
-      <ScrollToTop />
-      <div className="p-1 text-center">
-        <h1 className="mb-5 inline-flex items-center pt-3 text-2xl font-bold text-gray-900 dark:text-white sm:text-2xl md:text-3xl lg:text-4xl xl:text-5xl">
-          A Huge Thank You to Our Contributors
-        </h1>
-        <h3 className="pb-8 text-base font-light text-muted-foreground lg:text-lg">
-          Want to also be in this list? Contact us in the community chat and win
-          eternal glory!
-        </h3>
-        <ul>
-          <div className="justify-center xl:flex">
-            <div
-              id="grid-container"
-              className="w450:grid-cols-1 w450:gap-7 grid w-full grid-cols-2 gap-10 md:grid-cols-3 xl:w-1/2"
-            >
+    <>
+      <link
+        rel="stylesheet"
+        href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.7.0/css/all.min.css"
```

Comment:
Pretty sure global css injection should be done in <head> (i.e. top of application). Also why is it needed? Why not simply use the font-awesome components?